model/task_lifecycle.go

```diff
@@ -732,6 +732,13 @@ func tryDequeueAndAbortCommitQueueVersion(t *task.Task, cq commitqueue.CommitQue
 	}))
 
 	removed, err := cq.RemoveItemAndPreventMerge(issue, true, caller)
+	grip.Debug(message.Fields{
+		"message": "removing commit queue item",
+		"issue":   issue,
+		"err":     err,
```

Comment:
message.WrapError, unless there's a reason you want it to log either way

Reply:
I do want to log regardless of if there's an error